backend/dal/Repositories/AcquisitionFilePropertyRepository.cs

```diff
@@ -79,7 +79,8 @@ public PimsPropertyAcquisitionFile Update(PimsPropertyAcquisitionFile propertyAc
         {
             propertyAcquisitionFile.ThrowIfNull(nameof(propertyAcquisitionFile));
 
-            this.Context.Entry(propertyAcquisitionFile).CurrentValues.SetValues(propertyAcquisitionFile);
+            Context.Entry(propertyAcquisitionFile).CurrentValues.SetValues(propertyAcquisitionFile);
+            Context.Entry(propertyAcquisitionFile).State = EntityState.Modified;
```

Comment:
Need to tell EF to track changes for this entity because we get them in a disconnected state - `.AsNoTracking()` from the repo